### DISCUSSION TITLE
useRowState to dinamically hide/unhide cells and to embed additional content

### DISCUSSION THREAD

**Discussion by rodbs** [TanStack/table#2574] [General]
I'd like to programmatically manipulate each cell to create audio tables.
Initially I want to load some column totally empty (hide content)
Then when clicking on each cell individually I'd like unhide the content and to play a mp3 (which url would be embedded int the cell content, apart from the text)

Is it possible to build this with react-table? I've been reading the useRowState hook doc, but I don't understand  very well how to use it, and I haven't seen any examples about it.

Any ideas o examples would be highly appreciated it.
Thx

**Comment by johachi** [comment]
Sounds possible, but I'm not sure exactly what you want to do. Maybe you could share a [codesandbox](https://codesandbox.io) of what you got so far?
